main.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+require_relative 'app'
+
+def main
+  puts "Welcome to School Library App!\nPlease choose an option by enterin a number from 1 to 7:
+    \n1 - List all books\n2 - List all people\n3 - Create a person\n4 - Create a book
+    \n5 - Create a rental\n6 - List all rental for a given id\n7 - Exit"
+  action = gets.chomp.to_i
+
+  if action < 7 && action.positive?
+    start(action)
+  elsif action == 7
+    puts 'Good Bye!'
+  else
+    main
+  end
+end
+
+def start(action)
+  app = App.new
+  case action
+  when 1
+    app.list_all_books
+  when 2
+    app.list_all_people
+  when 3
+    app.create_a_person
+  when 4
+    app.create_a_book
+  when 5
+    app.create_student
+  when 6
+    app.list_all_rentals
+  end
+end
```

Comment:
- [ ] Your application doesn't run correctly, I'm running `ruby main.rb`  after cloning the app, here is the error message I get 👇🏻 it looks like your `attr_reader :classroom` in the student.rb file has some issues. Kindly correct this and ask for a review

![Screenshot from 2023-01-17 18-11-11](https://user-images.githubusercontent.com/79058364/212952945-5a3157dc-8ff9-4d89-b4aa-0f36c4040ec9.png)